code/lib/cli/rendererAssets/common/Configure.mdx

```diff
@@ -0,0 +1,311 @@
+import { Meta } from "@storybook/blocks";
```

Comment:
Shouldn't we rename the file to indicate, that it is a template? (use another file extension or whatsoever?)